src/main/scala/codechicken/microblock/ItemMicroBlock.scala

```diff
@@ -103,14 +103,19 @@ object ItemMicroBlock {
         stack.getOrCreateTag()
         if (!stack.getTag.contains("factory_id")) {
             //Wut..
-            logger.error("Found stack with no factory_id tag? {}", stack)
             -2000
```

Comment:
Im fine with changing this constant to -1, it makes a bit more sense. The `//Wut..` comment can also be removed

Reply:
Updated as requested.